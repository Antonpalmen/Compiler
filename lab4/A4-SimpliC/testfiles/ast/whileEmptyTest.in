int a() {
  int x;
  int y;
    while (x < y) {
    }
}
