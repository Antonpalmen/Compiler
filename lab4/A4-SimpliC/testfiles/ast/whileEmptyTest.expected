Program
  List
    Function
      IdDecl ID="a"
      List
      List
        Declaration
          IdDecl ID="x"
          Opt
        Declaration
          IdDecl ID="y"
          Opt
        WhileStatement
          LesserThan
            IdUse ID="x"
            IdUse ID="y"
          List
